### ISSUE TITLE
Add visual programmatic labels and instructions on Explore and Query pages

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2753]
As per Trusted Tester: 

- The combination of the accessible name, accessible description, and other programmatic associations (e.g., table column and/or row associations) describes each input field and includes all relevant instructions and cues (textual and graphical).
- Visual labels or instructions are provided for form elements.

[Explore Data](https://revenuedata.doi.gov/explore?dataType=Revenue&location=NF%2CNA&mapLevel=State&offshoreRegions=false&period=Fiscal%20Year&year=2022)

- Add message that only 4 locations can be selected. This needs to be both visible and available to screen readers at the point of the function. The visible message is current too discrete, removed from the interactive function, and there is no programmatic message attached to the add location functions.
- Year slider needs a visual label

- Year slider gets ANDI aria label and screen reader error
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdThFIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--b9d1be6ed16e834d0752a43e29764680997414f5/image.png)

- Dropdowns have no programmatic label or name
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdlVFIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--70949fdb0bf6f7daf65047c6a51e57d0eefa2680/image.png)

[Query Data](https://revenuedata.doi.gov/query-data)

- Year slider needs a visual label
- Search Box - Input type for the site search box is search, but the input type for the search companies box is text - is one more correct than the other?
- Data Type, period, and table filters function but are not programmatically labeled as fly-out/drop-down menus or 
collapsed/expanded

**Comment by AlexandracmONRR** [comment]
Search Box: Missing form label
Found in WAVE.

![image](https://github.com/DOI-ONRR/nrrd/assets/122935097/76aa71f1-8173-4821-a8a2-5c788805cf80)

**Comment by lpgoldstein** [comment]
No longer relevant
